### PR TITLE
Added AmazonSlider

### PR DESCRIPTION
**Pull request by SamarthKadam**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes:  #786 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

@VidhiBhatt01  Please see to it
# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK#Aldogni

### CODE REVIEW

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
The navbar is getting cut. I can't accept it. You can check out videos on how to make a hamburger navbar on YouTube as well as take help from someone on the server.
image

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-817--cool-front-end-templates.netlify.app/amazonhomepageslider/

File: AmazonHomePageSlider/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.google.com/icons?selected=Material%20Icons%3Aarrow_drop_down%3A+Filled">
+    <title>Document</title>
```
Comment (VidhiBhatt01) [inline AmazonHomePageSlider/index.html:9]: Add a suitable title

**Review by VidhiBhatt01 — COMMENTED** [review]
Fix this too

**Comment by SamarthKadam** [comment]
@VidhiBhatt01  Please review this

**Review by DEEPIKA-SUK — CHANGES_REQUESTED** [review]
@SamarthKadam add proper images in slider in mobile view, blurred images is not looking good.

**Review by VidhiBhatt01 — COMMENTED** [review]
I agree with deepali, rest all is awesome

**Comment by SamarthKadam** [comment]
@DEEPIKA-SUK  I've mentioned before it was done considering the project built for Desktop. If you see official amazon page it is not that responsive. I can't make the images aspect ratio change.Only thing I can say is  I recommend viewers  to view on desktop mode

**Review by VidhiBhatt01 — APPROVED** [review]
No worries :) Thanks for contributing to our repo!

**Comment by SamarthKadam** [comment]
Hoping Level 3 badge :)

**Review by arpit456jain — CHANGES_REQUESTED** [review]

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**